BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -88,11 +87,15 @@ fun SearchBar(
                 contentDescription = null
             )
         },
-        colors = TextFieldDefaults.textFieldColors(
-            backgroundColor = MaterialTheme.colors.surface
+        colors = TextFieldDefaults.colors(
+            unfocusedContainerColor = MaterialTheme.colorScheme.surface,
+            focusedContainerColor = MaterialTheme.colorScheme.surface
         ),
         placeholder = {
-            Text(stringResource(R.string.placeholder_search))
+            Text(
+                text = stringResource(R.string.placeholder_search),
+                style = MaterialTheme.typography.bodyLarge
```

Comment:
Compared to the M2 version, you're adding a custom style here. This will probably not be explained in the tutorial, so let's make sure we actually need it.

Reply:
it is slightly different without the custom style but not that noticeable